src/components/Questionnaire/QuestionTypes/SymptomQuestion.tsx

```diff
@@ -217,6 +216,99 @@ const SymptomRow = React.memo(function SymptomRow({
 
   const handleRemove = useCallback(() => onRemove(index), [index, onRemove]);
 
+  const datePickerField = (
+    <CombinedDatePicker
+      value={
+        symptom.onset?.onset_datetime
+          ? new Date(symptom.onset.onset_datetime)
+          : undefined
+      }
+      onChange={handleDateChange}
+      disabled={disabled || (!isSymptomInSheet && !!symptom.id)}
+      dateFormat="P"
+      buttonClassName="h-8 md:h-9 w-full justify-start font-normal"
+    />
+  );
+
+  const statusSelect = (
+    <Select
+      value={symptom.clinical_status}
+      onValueChange={handleStatusChange}
+      disabled={disabled}
+    >
+      <SelectTrigger className="h-8 md:h-9">
+        <SelectValue />
+      </SelectTrigger>
+      <SelectContent>
+        {SYMPTOM_CLINICAL_STATUS.map((status) => (
+          <SelectItem key={status} value={status}>
+            {t(status)}
+          </SelectItem>
+        ))}
+      </SelectContent>
+    </Select>
+  );
+
+  const severitySelect = (
+    <Select
+      value={symptom.severity}
+      onValueChange={handleSeverityChange}
+      disabled={disabled}
+    >
+      <SelectTrigger className="h-8 md:h-9">
+        <SelectValue />
+      </SelectTrigger>
+      <SelectContent>
+        {SYMPTOM_SEVERITY.map((severity) => (
+          <SelectItem key={severity} value={severity}>
+            {t(severity)}
+          </SelectItem>
+        ))}
+      </SelectContent>
+    </Select>
+  );
+
+  const notesInput = (
+    <Input
+      type="text"
+      placeholder={t("additional_notes")}
+      value={symptom.note || ""}
+      onChange={handleNotesChange}
+      disabled={disabled}
+    />
+  );
+
+  if (isSymptomInSheet) {
+    return (
+      <div className="space-y-3">
+        <div>
+          <div className="text-sm font-medium text-gray-700 mb-1">
+            {t("onset_date")}
+          </div>
+          {datePickerField}
+        </div>
+        <div>
+          <div className="text-sm font-medium text-gray-700 mb-1">
+            {t("status")}
+          </div>
+          {statusSelect}
+        </div>
+        <div>
+          <div className="text-sm font-medium text-gray-700 mb-1">
+            {t("severity")}
+          </div>
+          {severitySelect}
+        </div>
+        <div>
+          <div className="text-sm font-medium text-gray-700 mb-1">
+            {t("note")}
+          </div>
+          {notesInput}
+        </div>
+      </div>
+    );
+  }
```

Comment:
let's not assign components to variables inside a component's definition!

React will not be able to memoize things correctly and cause unnecessary re-renders due to change in reference on every render!